GUI/QtQuick/QtQuickAdapter.cpp

```diff
@@ -2079,12 +2086,13 @@ int QtQuickAdapter::changePassword(const QString& walletId, const QString& oldPa
    return (msgId == 0) ? -1 : 0;
 }
 
-int QtQuickAdapter::exportWalletAuth(const QString& walletId, const QString& password)
+int QtQuickAdapter::exportWallet(const QString& walletId)
```

Comment:
You're missing the output path param (ExportDir) here. It can be different than a setting value.